use-cases/rag-on-gke/backend/src/requirements.txt

```diff
@@ -0,0 +1,5 @@
+fastapi
+pydantic
+uvicorn
+typing
+llama-index==0.11.21
```

Comment:
Can you alphabetize these and pin them to a version?

Reply:
fixed